### DISCUSSION TITLE
How to block a e-mail adress from sending e-mails to all my aliases?

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1419] [Q&A]
I want a block a specific e-mail adress from sending e-mails to all my aliases. I have tried one-click unsubscribe but that's doesn't block the send of the e-mail that only disables my alias. I have changed the one-click unsubscribe button to "block the sender that send the original email" but also doesn't work. I saw a post from a few ago saying that the feature "Disabled alias/Blocked contact" doesn't work maybe it has something to do with that.

**Comment by McKorleone** [comment]
Go to the Web Dashboard and search for your Alias.
Click on Contacts and search for the specific e-mail which is bothering you.
Move the switch and done ;)

**Reply by gdonval** [reply]
I believe the problem is how to block an email address across all aliases/accounts, not just a single one. This is all the more relevant for catch-all domains.
